Misc/NEWS.d/next/Library/2019-05-06-22-34-47.bpo-33110.rSJSCh.rst

```diff
@@ -0,0 +1 @@
+Adds error handling to concurrent.futures add_done_callback, so that callbacks triggered on futures added to already completed futures are handled in the same manner as callbacks attached to running futures are, when the future subsequently finishes.
```

Comment:
That's a complicated sentence ;-) Maybe
```suggestion
Handle exceptions raised by functions added by concurrent.futures add_done_callback correctly when the Future has already completed.
```

Reply:
Thats much more succinct, will update it.